internal/gps/strip_vendor.go

```diff
@@ -12,10 +12,6 @@ import (
 )
 
 func stripVendor(path string, info os.FileInfo, err error) error {
-	if err != nil && err != filepath.SkipDir {
```

Comment:
yeah, finally looked at the `filepath.Walk()` implementation - `lstat()`, or `readDirNames()`. ok 😄